packages/sw-routing/test/automated-test-suite.js

```diff
@@ -49,7 +49,7 @@ describe(`sw-routing Browser Tests`, function() {
         .then(() => swTestingHelpers.mochaUtils.startWebDriverMochaTests(
           assistantDriver.getPrettyName(),
           globalDriverBrowser,
-          `${baseTestUrl}/test/browser-unit/`
+          `${baseTestUrl}/test/browser/`
```

Comment:
Pro this, but could we rename sw-unit to just sw rather than 'unit'?

My rationale for this originally was that some tests are unit tests in the browser, some are unit tests in sw and some service workers are essentially registered as assets to then test resulting behavior.

I've not be particularly standardised so would be good to get some consensus and strive to make all tests follow that pattern.